examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -1073,10 +1078,13 @@ def compute_time_ids(original_size, crops_coords_top_left):
                     loss = loss.mean(dim=list(range(1, len(loss.shape)))) * mse_loss_weights
                     loss = loss.mean()
 
+                for fname in batch['filenames']:
+                    accelerator.log({'loss_for_' + fname: loss}, step=global_step)
```

Comment:
Should we accept a CLI argument for this? That will be nice no?

Reply:
I agree, it's nice to have.